bin/autoproj_bootstrap

```diff
@@ -601,4 +601,4 @@ if !ops.skip_stage2?
     ops.call_stage2
 end
 ops.run_autoproj 'bootstrap', *bootstrap_options
```

Comment:
> and rescue somewhere?

Not sure it's really useful to rescue in this case, since you will get your non-zero exit status.

> No concerns about other methods using run_autoproj or backward compatibility issues?

v2 is still in RC, and any method that fails should raise (i.e. current behaviour is unexpected).